pyproject.toml

```
@@ -8,6 +8,9 @@ packages = [
     {include = "gardena_bluetooth"},
     {include = "gardena_bluetooth/py.typed"},
 ]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+]
```

The project uses poetry.
```suggestion
license = "MIT"
```
https://python-poetry.org/docs/pyproject/#license